kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -270,14 +269,14 @@ def cargo_bump_version(
     task = project.do(
         name,
         CargoBumpVersionTask,
-        group=group,
+        group=CARGO_BUILD_SUPPORT_GROUP_NAME,
```

Comment:
could you also add a comment explaining why `CARGO_BUILD_SUPPORT_GROUP_NAME` is needed? the explaination in your PR description about `env!("CARGO_PKG_VERSION")` would suffice